readme.md

```diff
@@ -2,9 +2,27 @@
 
 # WordPress Docker Development Environment
 
-This is a Docker based local development environment for WordPress.
-
-## What's Inside
+> This is a Docker based local development environment for WordPress.
+
+[![Support Level](https://img.shields.io/badge/support-active-green.svg)](#support-level)
```

Comment:
@jeffpaul License should be MIT on this

Reply:
done, see: https://github.com/10up/wp-local-docker/pull/142/commits/090d6e5e55ac0e8a025b9b63c3855864fbe02a40 && https://github.com/10up/wp-local-docker/pull/142/commits/df1abf1e4980eedc057addcca4f18996986cec0e